### PR TITLE
add hwi/oauth-bundle

### PR DESCRIPTION
**Pull request by andreybolonin**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: hwi/oauth-bundle/0.6/config/packages/hwi_oauth.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+hwi_oauth:
+    # list of names of the firewalls in which this bundle is active, this setting MUST be set
+    firewall_names: [secured_area]
+
+    # an optional setting to configure a query string parameter which can be used to redirect
+    # the user after authentication, e.g. /connect/facebook?_destination=/my/destination will
+    # redirect the user to /my/destination after facebook authenticates them.  If this is not
```
Comment (Nyholm) [inline hwi/oauth-bundle/0.6/config/packages/hwi_oauth.yaml]: Do we really need all these comments and configuration also as a comment? I would like to see a link to the documentation instead. 

Also, remove all "inactive" configuration

File: hwi/oauth-bundle/0.6/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<bg=blue;fg=white>                                          </>
+<bg=blue;fg=white> What's next for HWIOAuthBundle? </>
+<bg=blue;fg=white>                                          </>
+
+  * <fg=blue>Configure</> your application:
+    1. Enable sessions by uncommenting the entire session section in <comment>config/packages/framework.yaml</>
```
Comment (Nyholm) [inline hwi/oauth-bundle/0.6/post-install.txt]: This instruction is not really reliable. Maybe write exactly what to do here or refer to documentation. 

> Enable sessions by adding `framework.session.enabled: true`  in <comment>config/packages/framework.yaml

**Comment by andreybolonin** [comment]
@Nyholm done

File: hwi/oauth-bundle/0.6/config/packages/framework.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+framework.session.enabled: true
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply hwi/oauth-bundle/0.6/config/packages/framework.yaml]: We should not modify other bundles configuration. 

Please remove this. 
If it is really needed, write something in the post-install.

**Comment by Nyholm** [comment]
Good. I like this. 

Could we provide a better example? When I try to install this I get the following error. 
screen shot 2018-06-22 at 18 06 34


Maybe we add "facebook" as default, What do you think?

**Comment by andreybolonin** [comment]
@Nyholm yep

File: hwi/oauth-bundle/0.6/config/packages/hwi_oauth.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+hwi_oauth:
+    # list of names of the firewalls in which this bundle is active, this setting MUST be set
+    firewall_names: [secured_area]
```
Comment (Nyholm) [inline hwi/oauth-bundle/0.6/config/packages/hwi_oauth.yaml]: I like this comment. But may I suggest to change this value to "main"? The default security config looks like: 

```
security:
    # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
    providers:
        in_memory: { memory: ~ }
    firewalls:
        dev:
            pattern: ^/(_(profiler|wdt)|css|images|js)/
            security: false
        main:
            anonymous: true

            # activate different ways to authenticate

            # http_basic: true
            # https://symfony.com/doc/current/security.html#a-configuring-how-your-users-will-authenticate

            # form_login: true
            # https://symfony.com/doc/current/security/form_login_setup.html

    # Easy way to control access for large sections of your site
    # Note: Only the *first* access control that matches will be used
    access_control:
        # - { path: ^/admin, roles: ROLE_ADMIN }
        # - { path: ^/profile, roles: ROLE_USER }

```

There is no firewall named "secured_area". What do you think?

**Comment by andreybolonin** [comment]
@Nyholm thank you for review )
what about to merge?

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
The review bot seams to be on vacation. I'll be back when it is fixed.

**Comment by andreybolonin** [comment]
@symfony-flex-server review please

**Comment by fabpot** [comment]
@symfony-flex-server review please (hey, that's me, be nice :))

**Pull request merged**